source/localizable/tutorial/subroutes.md

```diff
@@ -0,0 +1,398 @@
+
+```js
+let route = this.subject({ store });
+```
+
+Finally, we call the `model` function directly, which will trigger the assertions we wrote on our stubbed `store` object.
+
+```js
+route.model({ slug: 'rental-thing' });
+```
+
+# Adding the Rental To Our Template
+
+Next, we can update the template for our show route (`app/templates/rentals/show.hbs`) and list the information for our rental.
+
+```app/templates/rentals/show.hbs
+<div class="jumbo">
+  <a href="#" class="button right up-11">Edit</a>
+  <h2 style="margin-bottom:15px">{{model.title}}</h2>
+  <div class="right" style="width:50%;padding-left:30px">
+    <div class="map right" style="background: url({{model.address}}) center center;"></div>
+    <div class="detail" style="margin-top:10px">
+      <strong>Owner:</strong> {{model.owner}}
+    </div>
+    <div class="detail">
+      <strong>Type:</strong> {{rental-property-type model.type}} - {{model.type}}
+    </div>
+    <div class="detail">
+      <strong>Location:</strong> {{model.city}}
+    </div>
+    <div class="detail">
+      <strong>Number of bedrooms:</strong> {{model.bedrooms}}
+    </div>
+    <p>&nbsp;</p>
+    <p>{{model.description}}</p>
+  </div>
+  <img src="{{model.image}}" alt="" class="" style="width:50%;height:initial;position:static">
+</div>
+```
+
+We pass our `model` to the `rental-listing` component and add a new section listing the rental's `title`.
+Browse to `localhost:4200/rentals/grand-old-mansion` and you should see the information listed for that specific rental.
+
+# Adding an Index Sub-route
+
+One of the first things you may notice is that if you browse directly to `localhost:4200/rentals`,
+all we see are the items from our parent route (`app/template/rentals.hbs`).
+This is because we do not have an `index` sub-route.
+Much like how our application defaults to the `index` route,
+our `rentals` route defaults to `index` and tries to render it in our `outlet`.
+
+We will want to generate an `index` route and use it to list all of our rentals and allow the user to select one to view additional information.
+
+```shell
+ember g route rentals/index
+```
+
+If you open up your Router (`app/router`) you may notice that nothing has updated.
+
+```app/router.js
+Router.map(function() {
+  this.route('about');
+  this.route('contact');
+  this.route('rentals', function() {
+    this.route('show', { path: '/:slug' });
+  });
+});
+```
+
+Much like how our applications `index` route doesn't appear in our Router, `index` routes on sub-routes won't explicitly appear in the Router either.
+Ember knows that the default action is to take the user to the `index` route.
+However, you can modify the `index` route if needed.
+For example, you can modify the `path` much like we do with our `show` route by adding `this.route('index', { path: '/custom-path'})`.
+
+Next, let's modify the our `index` route.
```

Comment:
Great point, and I believe you had brought that up before. Let me know what you think the best route is. I feel it may be best to address it here and not alter the existing tutorials (for the edge cases of people who have already started going through it)